app/Models/Presentation.php

```diff
@@ -123,13 +123,30 @@ public function speakers(): Attribute
         );
     }
 
+    /**
+     * Returns a speaker of the presentation that was registered first,
+     * and therefore had most likely created the presentation
+     *
+     * @return Attribute
+     */
+    public function firstSpeaker(): Attribute
+    {
+        return Attribute::make(
+            get: fn() => User::whereHas('userPresentations', function ($query) {
+                $query->where('presentation_id', $this->id)
+                    ->where('role', 'speaker');
+            })->orderBy('created_at')
+                ->first()
+        );
+    }
+
```

Comment:
Maybe `creator` would be a better name? Idk not mandatory to change just saying it might have better name.